src/data/markdown/translated-guides/en/02 Using k6/00 Tutorial/300 Analyze results.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Analyze results
+excerpt: Use k6 to write custom metrics and filter results.
+---
+
+The end-of-test summary provides only an overview of performance.
+In real analysis, it helps to be able to analyze the test along data points and for specific requests.
+
+In this tutorial, learn how to:
+- Use tags to filter results
+- Organize requests in groups
```

Comment:
I would recommend most users to just use the docker-compose setup with influxdb + grafana. 

It is way more user friendly and while it won't scale quite a lot , for this example it is overkill. 

But will be way easier then using jq and will scale the UX way better.

The downside is that it has it's own problems and we are slowly trying to move away from it.

But the usage of `jq` while nice is likely way too technical for most users, and while I love `jq` it error messages are as cryptic as possible IME.

Reply:
I completely agree with your sentiment, as `jq` is on par with `ffmpeg` for being highly useful, highly used, and highly confusing. But `jq` is really portable so it can be used everywhere, and the other implementation, as you mentioned, may be going. 

My initial plan was to use node, but then I discovered #1191. Having little preprocessors to format the output or to use handlesummary seemed like more complexity than it was worth. And visualization is a whole other category, but these docs to link out to relevant places a few times.

Still I doubt `jq` is the best option, but it's the simplest one I can think of.